src/tests/ftest/control/dmg_telemetry_io_latency.py

```diff
@@ -315,13 +315,16 @@ def verify_io_dtx_committed_metrics(self, baseline_data, metrics_data, test_metr
         # test ops = (create dir + punch obj + punch dir) * repetitions
         test_ops *= repetitions
         num_operations += test_ops
+        # 2 DTX entries on 2 replicas
+        test_servers = self.params.get("test_servers", "/run/*")
+        num_operations *= test_servers
```

Comment:
This caused the CI test failure:
05:02:17 ERROR| TypeError: can't multiply sequence by non-int of type 'float'